pandas/core/indexes/base.py

```diff
@@ -122,6 +122,29 @@ class Index(IndexOpsMixin, PandasObject):
     Notes
     -----
     An Index instance can **only** contain hashable objects
+
+    Examples
+    --------
+    >>> pd.Index([1, 2, 3])
+    Int64Index([1, 2, 3], dtype='int64')
+
+    >>> pd.Index(list('abc'))
+    Index(['a', 'b', 'c'], dtype='object')
+
+    See Also
+    ---------
+    RangeIndex : Index implementing a monotonic integer range (very
```

Comment:
I think that most of these are self-describing, so you can leave off the description. I would keep the descriptions for

- RangeIndex
- MultiIndex
- IntervalIndex
- CategoricalIndex (maybe)

The rest can be put on a couple lines likes

```
DatetimeIndex, PeriodIndex, TimedeltaIndex
Int64Index, UInt64Index, Flaot64Index
```

since the names tell you pretty much everything you need to know.

Reply:
Ok, done.